CMakeLists.txt

```diff
@@ -352,7 +410,6 @@ list(APPEND SUNSHINE_EXTERNAL_LIBRARIES
 		libminiupnpc-static
 		${CBS_EXTERNAL_LIBRARIES}
 		${CMAKE_THREAD_LIBS_INIT}
-		stdc++fs
```

Comment:
hum...can someone check if this is the reason, why the Linux and Windows builds fail?

Reply:
What is interesting is the AppImage is building. Based on that I'm guessing it would still build on the Ubuntu/Debian Dockerfiles would still build. I have fast fail on for the Dockerfiles build job.

Reply:
The strange thing is that this is the only change that applies also to none Apple platforms. Everything else should only be active with the MacOS build. I think I have to setup a Linux VM on the weekend and debug that in more detail.

Reply:
My only guess is that `if(!APPLE)` is not working as it should and Boost is trying to linking itself as a dynamic library instead of a static one.  Isn't the correct definition `if(NOT APPLE)` ?

Reply:
> My only guess is that `if(!APPLE)` is not working as it should and Boost is trying to linking itself as a dynamic library instead of a static one. Isn't the correct definition `if(NOT APPLE)` ?
> 
> EDIT: At least according to https://cmake.org/cmake/help/v3.3/command/if.html

I agree although not an expert on CMake, I am basing my opinion on the fact the errors seem to be related to boost.

Reply:
Since we also use `NOT` instead of `!` further down below in the file, I would say it is safe to assume that `NOT` is the right way to do it. I will push an update in a second.

Reply:
Looks like that did it 😊